vraptor-site/content/en/docs/one-minute-guide.html

```diff
@@ -6,7 +6,7 @@
 
 We will call a `Controller` classes that contain the business model of your application. Those are classes commonly called actions or even services by other frameworks, although they don't mean exactly the same.
 
-With VRaptor's jar in your project, create our controllers in order to receive requests and start building your project.
+With VRaptor's jar and its dependencies in your project, create our controllers in order to receive requests and start building your project.
```

Comment:
It sounds nice! 
What do you think about adding a link to dependencies page here?

> With VRaptor's jar and [its dependencies](link to dependencies page here) in your project, etc etc